lib/CohereEmbedV3ServicesStackProps.ts

```diff
@@ -52,6 +52,10 @@ export interface CohereEmbedV3Services {
      * The memory type for the deployment.
      */
     readonly cdkDeployMemoryType: string;
+    /**
+     * The vpcId for the deployment.
+     */
+    readonly vpcId: string;
```

Comment:
**Suggestion:** The `vpcId` property is added to the `CohereEmbedV3Services` interface, but it should also be added to the `CohereEmbedV3ServiceArgs` interface for consistency.
> 📋  **Medium Priority** 
 > [maintainability, importance: 6]
```suggestion
    readonly vpcId: string;
    /**
     * The vpcId for the deployment.
     */
    readonly vpcId: string;
```